builtin/logical/database/rotation_test.go

```diff
@@ -81,7 +81,6 @@ func TestBackend_StaticRole_Rotate_basic(t *testing.T) {
 	}
 
 	data = map[string]interface{}{
-		"name":                "plugin-role-test",
```

Comment:
This was causing a test failure because the test expected an empty response. The presence of "name" would trigger a response with warnings for replaced parameters because the path already provides the name of the role.